tensorflow-core/tensorflow-core-api/src/bazel/api_def/import/api_import.cc

```diff
@@ -0,0 +1,317 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Could this be translated to Java? And if not, why not?

Reply:
Like I mentioned in the description of this PR, this is just an old utility I've written awhile ago when classifying the 900+ initial operation wrappers and I just put it there so it is not lost, as I still find it quite useful.

Now could it be rewritten in Java... I guess so, but with a lot of work as it depends on TF internal classes. I'm not planning to spend time on this anytime soon so if we think it does not fit in our repo, then I'll simply remove it from this PR and continue to keep it in my fork.

Reply:
Does this code actually gets used during the build? If so, I suppose we can leave it here...

Reply:
No, I only use it when I discover that new operations have been added after a TF upgrade, so I can classify them properly by configuring the [API definitions](https://github.com/karllessard/tensorflow-java/tree/master/tensorflow-core/tensorflow-core-api/src/bazel/api_def) properly before rebuilding. It just gives me some hints in which package they should be located based on Python endpoints that has a similar name.

Reply:
Ah, ok, so let's add the step to compile it to the build.sh script or something while we're at it?

Reply:
It's already there, see [here](https://github.com/karllessard/tensorflow-java/blob/d7618c2b9be4b5f0d4942ef8adbd571e8aa6bc43/tensorflow-core/tensorflow-core-api/build.sh#L37) and [here](https://github.com/karllessard/tensorflow-java/blob/d7618c2b9be4b5f0d4942ef8adbd571e8aa6bc43/tensorflow-core/tensorflow-core-api/BUILD#L62)